src/BitmovinEncodingLambda/index.js

```diff
@@ -1,3 +1,32 @@
+import mysql from 'mysql';
+
+function storeBitmovin(input, folder) {
+  const livestream_dir = 'bitmovin/' + folder
+  console.log('livestream_dir', livestream_dir)
+  console.log(`input: ${input}`);
+  updateAsset(input, livestream_dir);
+}
+
+function updateAsset(input, livestream_dir) {
+  const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_DATABASE,
+  });
+
+  connection.connect();
+
+  const query = 'UPDATE content_library_assets SET livestream_dir = ?  WHERE tmp_video = ? ';
+  console.log(`query: ${query}`);
+
+  connection.query(query, [livestream_dir, input], (err, result) => {
+    if (err) throw err;
```

Comment:
yes, we should add it. it would need to be a cloudwatch notifications. maybe add a separate card for this?